templates/login.tpl

```diff
@@ -10,7 +10,7 @@
 				<p>{error}</p>
 			</div>
 
-			<form class="form-horizontal" role="form" method="post" target="login" id="login-form">
+			<form class="form-horizontal" role="form" method="post" id="login-form">
```

Comment:
No, the current use of `target` violates the standard because this is instructing the browser to open the _response_ on an `iframe` named _target_. This does not make any sense as there's no frame on the login page.

Use `action` to send to the same address we already are is redundant. Without `action` the POST is sent to the same current page, which is _login_. `action` is only needed if you want the request to be done in another document other than the current, like `download_file`.